lib/commons/aria/get-role.js

```diff
@@ -158,6 +165,7 @@ function resolveRole(node, { noImplicit, ...explicitRoleOptions } = {}) {
  * @param {boolean} options.abstracts  Allow role to be abstract
  * @param {boolean} options.dpub  Allow role to be any (valid) doc-* roles
  * @param {boolean} options.noPresentational return null if role is presentation or none
+ * @param {boolean} options.includeChromiumRoles Include implicit roles from chromium-based browsers in role result
```

Comment:
This looks a little out of place with the rest. Just calling it `.options.chormium` seems like it would be more consistent with the other options here.